backend/src/main/java/com/woowacourse/kkogkkog/reservation/application/ReservationService.java

```diff
@@ -1,109 +1,101 @@
-
-    public void update(ReservationUpdateRequest request) {
-        Member loginMember = findMember(request.getMemberId());
-
-        Reservation reservation = findReservation(request.getReservationId());
-        reservation.changeCouponStatus(CouponEvent.of(request.getEvent()), loginMember);
-
-        Coupon coupon = reservation.getCoupon();
-        MemberHistory memberHistory = saveMemberHistory(coupon.getOppositeMember(loginMember),
-            loginMember, coupon, CouponEvent.of(request.getEvent()), reservation.getMeetingDate(),
-            request.getMessage());
-
-        if (validateCancelReservation(request)) {
-            reservationRepository.delete(reservation);
-            coupon.updateMeetingDate(null);
-        }
-
-        publisher.publishEvent(PushAlarmEvent.of(memberHistory));
-    }
-
-    private boolean validateCancelReservation(ReservationUpdateRequest request) {
-        return request.getEvent().equals("CANCEL") || request.getEvent().equals("DECLINE");
-    }
-
-    private MemberHistory saveMemberHistory(Member member, Member loginMember,
-                                            Coupon coupon, CouponEvent request,
-                                            LocalDateTime dateTime, String message) {
-        MemberHistory memberHistory = new MemberHistory(null, member, loginMember,
-            coupon.getId(), coupon.getCouponType(), request, dateTime, message);
-        memberHistoryRepository.save(memberHistory);
-        return memberHistory;
-    }
-
-    private Reservation findReservation(Long reservationId) {
-        return reservationRepository.findById(reservationId)
-            .orElseThrow(ReservationNotFoundException::new);
-    }
-}
+//package com.woowacourse.kkogkkog.reservation.application;
+//
+//import com.woowacourse.kkogkkog.coupon2.domain.repository.CouponRepository;
+//import com.woowacourse.kkogkkog.infrastructure.event.PushAlarmEvent;
+//import com.woowacourse.kkogkkog.member.domain.Member;
+//import com.woowacourse.kkogkkog.member.domain.MemberHistory;
+//import com.woowacourse.kkogkkog.member.domain.repository.MemberHistoryRepository;
+//import com.woowacourse.kkogkkog.member.domain.repository.MemberRepository;
+//import com.woowacourse.kkogkkog.member.exception.MemberNotFoundException;
+//import com.woowacourse.kkogkkog.reservation.application.dto.ReservationSaveRequest;
+//import com.woowacourse.kkogkkog.reservation.application.dto.ReservationUpdateRequest;
+//import com.woowacourse.kkogkkog.reservation.domain.Reservation;
+//import com.woowacourse.kkogkkog.reservation.domain.repository.ReservationRepository;
+//import com.woowacourse.kkogkkog.reservation.exception.ReservationNotFoundException;
+//import java.time.LocalDateTime;
+//import org.springframework.context.ApplicationEventPublisher;
+//import org.springframework.stereotype.Service;
+//import org.springframework.transaction.annotation.Transactional;
+//
+//@Transactional
+//@Service
+//public class ReservationService {
```

Comment:
이런식으로 TODO 주석을 추가하면 인텔리제이의 TODO 탭을 통해 이런 파일들을 쉽게 관리할 수 있을 것 같아요. 

```suggestion
// TODO: should be deleted
//@Transactional
//@Service
//public class ReservationService {
```